src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -26,8 +30,8 @@ import { ReactionPage } from "components/templates/ReactionPage";
 import { ChatSidebar } from "components/organisms/ChatSidebar";
 import { UserProfileModal } from "components/organisms/UserProfileModal";
 import { WithNavigationBar } from "components/organisms/WithNavigationBar";
-
```

Comment:
We have a newline of space between templates, organisms, molecules, etc. So should keep this here.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1492/commits/516bcef90616e538cbf0a8609a96d134bfbf4fcc